toolkit/scss/forms/_summary.scss

```diff
@@ -46,7 +49,7 @@
 
 .summary-item-no-content {
   color: $secondary-text-colour;
-  @include core-16;
+  @include copy-16;
```

Comment:
`copy-16` brings in `padding-top` and `padding-bottom` so do we still need those 4 lines down from here?